src/client/components/ProductsList/actions.js

```diff
@@ -2,22 +2,37 @@ import { ENDPOINT_URL } from '../../common/variables';
 
 // Action types
 export const TOGGLE_ITEM = 'TOGGLE_ITEM';
+export const VOTE_FOR_ITEM = 'VOTE_FOR_ITEM';
 
 // Action creators
 const toggleItem = item => ({ type: TOGGLE_ITEM, item });
+const voteForItem = item => ({ type: VOTE_FOR_ITEM, item });
 
-// Dispatchers
-const toggle = (id, isOrdered) => dispatch => {
+export const toggle = (id, isOrdered, updatedAuthor) => dispatch => {
   fetch(`${ENDPOINT_URL}/item/${id}/update`, {
-    body: JSON.stringify({ _id: id, isOrdered: !isOrdered }),
+    body: JSON.stringify({
+      _id: id,
+      isOrdered: !isOrdered,
+      author: updatedAuthor
+    }),
     headers: {
       'Content-Type': 'application/json'
     },
     method: 'PATCH'
   })
     .then(resp => resp.json())
-    .then(item => setTimeout(() => dispatch(toggleItem(item)), 500))
+    .then(item => setTimeout(() => dispatch(toggleItem(item)), 600))
     .catch(err => console.error(err));
 };
 
-export default toggle;
+export const vote = (id, votes) => dispatch => {
+  const payload = { _id: id, votes };
+  fetch(`${ENDPOINT_URL}/item/${id}/update`, {
+    body: JSON.stringify(payload),
+    headers: { 'Content-Type': 'application/json' },
+    method: 'PATCH'
+  })
+    .then(resp => resp.json())
+    .then(item => dispatch(voteForItem(item)))
+    .catch(err => console.error(err));
```

Comment:
wouldn't be better to  `console error(err.message)` to get more specific message when error occurs?